mkdocs/sys-amin/docs/configure-tactic.md

```diff
@@ -40,7 +55,63 @@ files.
 </tr>
 <tr class="even">
 <td><p>INCLUDE_JS</p></td>
-<td><p>You can include one or more external js files you want to make use of separated by ,.</p></td>
+<td><p>You can include one or more external js files you want to make use of. Separated by ,.</p></td>
+</tr>
+<tr class="odd">
+<td><p>SERVER</p></td>
+<td><p>A prefix for code generation to distinguish data from different servers</p></td>
+</tr>
+<tr class="even">
+<td><p>INCLUDE_CSS</p></td>
+<td><p>You can include one or more external css files you want to make use of. Separated by ,.</p></td>
+</tr>
+<tr class="odd">
+<td><p>INSTALL_DIR</p></td>
+<td><p>The installation directory for TACTIC</p></td>
+</tr>
+<tr class="even">
+<td><p>SITE_DIR</p></td>
+<td><p>The directory storing user informations</p></td>
```

Comment:
This doesn't seem to be used. It's commented out in the source code 
./src/pyasm/common/environment.py:        #return Config.get_value("install","site_dir")
Can you verify and remove the commented out line?